### PR TITLE
Per issue #29 - Adding "id" field to credentialStatus in example

### PR DESCRIPTION
**Pull request by yanicakj**
Referencing this issue: https://github.com/w3c-ccg/vc-status-list-2021/issues/29

The example was a little misleading with the missing "id" field, adding it with this change.

***

### :boom: Error: connect ETIMEDOUT 140.82.113.6:443 :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jun 13, 2022, 7:51 PM UTC)_.

<details>
<summary>More</summary>

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c-ccg/vc-status-list-2021%2330.)._
</details>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by msporny — CHANGES_REQUESTED** [review]
The entry is a bit value in the list, so the suggested correction here is not correct. I've suggested changes here to something that might make a bit more sense.

File: index.html
Comment (msporny) [reply index.html:381]: Hmm, thinking more about this... I remember @dlongley and I had a conversation a while ago about the `id` value here and decided to not put one, but can't remember all of the arguments against... or if I'm just imagining that conversation at this point.

Comment (yanicakj) [reply index.html]: thanks for the correction 👍

**Review by msporny — APPROVED** [review]
LGTM.

**Review by OR13 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Comment by yanicakj** [comment]
Hey all, I'm unable to merge this change after approvals, is there any additional steps needed? Not sure how it works - if PRs are merged on weekly/monthly basis by your team

**Comment by msporny** [comment]
> Hey all, I'm unable to merge this change after approvals, is there any additional steps needed? Not sure how it works - if PRs are merged on weekly/monthly basis by your team

PRs are merged by Editors after enough review. This one has had enough review at this point, so I'll merge it down now.

If you'd like to become an Editor, please let us know and we can show you the ropes of what it takes to be an Editor of a specification.

**Pull request merged**
